projects/python-client/src/datapane/client/api/report/core.py

```diff
@@ -41,7 +41,7 @@
 local_post_xslt = etree.parse(str(local_report_def / "local_post_process.xslt"))
 local_post_transform = etree.XSLT(local_post_xslt)
 
-CDN_BASE = f"https://datapane-cdn.com/v{dp_version}"
+CDN_BASE = os.getenv("DATAPANE_CDN_BASE", f"https://datapane-cdn.com/v{dp_version}")
```

Comment:
should base include the version or just the URL? i.e. is it useful for us to be able to override the entire URL, including version, that the generate files use as their base?

genuinely unsure - happy to discuss and decide

Reply:
w.r.t. our docs: they only (officially) support the latest version of Datapane, a decision made at the beginning of our docs makeover! In that case, we want this to always reference the latest `dp_version`.

Reply:
The current structure allows for serving the statics yourself if you want.

ie, `DATAPANE_CDN_BASE=/statics/reports/`

Only reason we're not doing it with docs is we need to support screenshots, which are run in File mode, instead of via a Server (needed for JS imports)